infra/modular-examples/aws/main.tf

```diff
@@ -0,0 +1,553 @@
+terraform {
```

Comment:
I understand that this should be the only `main.tf` to exist for `aws` from now on